gem/templates/rwanda/profiles/viewprofile.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% load i18n wagtailsettings_tags %}
+{% get_settings %}
+
+{% block content %}
+  <div class="heading">
+    <h1 class="heading__profiles">
+      {% trans "Hello " %} {{ user.profile.migrated_username|default:user.username }}
+    </h1>
+  </div>
+  <div class="profiles">
+    <div class="profiles__account"
+    <fieldset>
+      <label class="profiles__description--label">{% trans "When you type comments your name will appear as:" %}</label>
+      <p class="profiles__description profiles__alias">{{ user.profile.alias | default:" Anonymous " }}</p>
+    </fieldset>
+
+    {% trans "Not set." as defaultText %}
+    <fieldset>
+      <label class="profiles__description--label">{% trans "Date of Birth: " %}</label>
```

Comment:
@KaitCrawford is there a specific reason the classes to modify profiles__description has different name? They could just use the same modifier class

Reply:
@Mitso I'm afraid I don't understand. Which modifier class should they be using? (tbh I'm not sure what a 'modifier class' is)